tests/mixed_int8/README.md

```diff
@@ -1,37 +1,122 @@
 # Testing mixed int8 quantization
 
+![HFxbitsandbytes.png](https://s3.amazonaws.com/moonup/production/uploads/1660567705337-62441d1d9fdefb55a0b7d12c.png)
+
+Hi there, this is a recipe on how to effectively debug `bitsandbytes` integration on HuggingFace `transformers`.
+
+## Library requirements
+
++ `transformers>=4.22.0`
++ `accelerate>=0.12.0` 
++ `0.31.8>=bitsandbytes>=0.31.5`.
 ## Hardware requirements
 
-I am using a setup of 2 GPUs that are NVIDIA-Tesla T4 15GB
+I am using a setup of 2 GPUs that are NVIDIA-Tesla T4 15GB - `younes-testing-multi-gpu` on GCP. To run successfully `bitsandbytes` you would need a 8-bit core tensor supported GPU. Note that Ampere architectures should be supported. Here is an exhaustive list of the supported GPU types at the time of this writing:
```

Comment:
I would avoid using the first person pronouns like `I am ...` in writing.

It would be more natural to be `The following instructions are tested with 2 NVIDIA-Tesla T4 GPUs`.

Reply:
```suggestion
The following instructions are tested with 2 NVIDIA-Tesla T4 GPUs. To run successfully `bitsandbytes` you would need a 8-bit core tensor supported GPU. Note that Ampere architectures should be supported. Here is an exhaustive list of the supported GPU types at the time of this writing:
```

Reply:
Thanks for the suggestion!

Reply:
It's also unnecessary to add `younes-testing-multi-gpu` on GCP`.
(This doesn't give any useful information to the readers)
(It's also best not to reveal information regarding internal infrastructure)

Reply:
Yep! Sorry for that 🙏

Reply:
Should fixed in 8e70e5f70384bead7009ad628a8e43f8ceb47d2e